packages/graphql/test/unit/entities/Wizard.spec.ts

```diff
@@ -1,188 +1,22 @@
-import type { FullConfig } from '@packages/types'
 import { expect } from 'chai'
 import { LocalProject, Wizard } from '../../../src'
 import { TestActions, TestContext } from '../../integration/utils'
 
-const createActionsWithResolvedConfig = (cfg: FullConfig) => {
+const createActionsWithResolvedConfig = () => {
   const Ctor = class Actions extends TestActions {
-    resolveOpenProjectConfig () {
-      return cfg
+    isFirstTime (projectRoot: string, testingType: Cypress.TestingType) {
+      return true
     }
   }
 
   return Ctor
 }
 
 describe('Wizard', () => {
-  describe('#shouldLaunchCt', () => {
```

Comment:
will write tests when we figure out final architecture

Reply:
Yeah let's hold off on unit tests for GQL for now.